### PR TITLE
Compliance work

### PR DESCRIPTION
**Pull request by dtaivpp**
This compliance work adds many of the core repo elements we would need to launch this as an official OpenSearch Project website. The last piece that is missing is work to create a footer that contains many of these compliance pages.

### CODE REVIEW

**Comment by dtaivpp** [comment]
This is in relation to #1

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Copyright OpenSearch Contributors.
+
+Copyright (c) Django Software Foundation and individual contributors.
```
Comment (timursaurus) [inline LICENSE]: I think we can remove this
and credit `vitepress` (Evan You)
https://vitepress.dev/

Comment (dtaivpp) [reply LICENSE]: Good catch, resolved.

**Comment by dtaivpp** [comment]
@timursaurus can you look over this again? Got the license updated

**Comment by timursaurus** [comment]
> @timursaurus can you look over this again? Got the license updated

LGTM!
Should we merge this now or when the footer is done?

**Comment by dtaivpp** [comment]
I feel like we should merge.  I am not sure what the footer work will take so probably good to split them out.

**Pull request merged**